lava-docker-slave-image/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM bitnami/minideb:stretch
+
+RUN apt-get update
+
+# e2fsprogs is for libguestfs
+RUN \
+ echo 'lava-server   lava-server/instance-name string lava-slave-instance' | debconf-set-selections && \
+ echo 'locales locales/locales_to_be_generated multiselect C.UTF-8 UTF-8, en_US.UTF-8 UTF-8 ' | debconf-set-selections && \
+ echo 'locales locales/default_environment_locale select en_US.UTF-8' | debconf-set-selections && \
+ DEBIAN_FRONTEND=noninteractive apt-get -y install \
+ locales \
+ vim \
+ sudo \
+ cu \
+ conmux \
```

Comment:
cu and conmux aren't strictly needed for the slave image, right?  Those should probably be in the overlay images.

Reply:
cu and conmux moved